tests/config.yaml

```diff
@@ -12,9 +12,9 @@ testdependecies:
         - nc
 module:
     docker:
-        start: "docker run -e MYHOSTNAME=localhost -p 25:10025"
+        start: "docker run -e MYHOSTNAME=localhost -p 10025:10025"
         source: https://github.com/container-images/postfix.git
-        container: docker.io/modularitycontainers/postfix
+        container: postfix
```

Comment:
MTF pulls the image specified in config.yaml. In this case postfix which should be mentioned in `docker images` no from docker hub. https://github.com/fedora-modularity/meta-test-family/blob/devel/moduleframework/helpers/container_helper.py#L146 If I understand right.